src/lib/override/adopted-probes.ts

```
@@ -80,79 +87,83 @@ export class AdoptedProbes {
 		},
 		hardwareDeviceFirmware: {
 			probeField: 'hardwareDeviceFirmware',
-			shouldUpdateIfCustomCity: true,
+			mayBeCustom: false,
 		},
 		systemTags: {
 			probeField: 'tags',
-			shouldUpdateIfCustomCity: true,
+			mayBeCustom: false,
 			formatter: (probeTags: Tag[], _probe: Probe, dProbe?: DProbe) => [
 				...(dProbe && dProbe.publicProbes ? [ this.getGlobalUserTag(dProbe.defaultPrefix!) ] : []),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)